core/templates/dev/head/css/oppia.css

```diff
@@ -672,6 +670,12 @@ body {
   color: white;
 }
 
+@media (max-width: 610px) {
+  .oppia-subscription-button {
+    margin: 0;
```

Comment:
The subscription button seems to have more space below the button than the space above the button (only in case of mobile). Adjust the position of the button so that it looks uniform. Maybe adding a margin-top with some value would do the work(again only in case of mobile).

Yours - > 
![image](https://user-images.githubusercontent.com/23234392/39330489-6f6e5ac0-49bf-11e8-8be5-27b1f9897a69.png)

Expected 
![screenshot from 2018-04-27 02-01-01](https://user-images.githubusercontent.com/23234392/39330547-a1aea45e-49bf-11e8-8282-3e65056586dc.png)